docs/setup/secure-settings.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[secure-settings]]
+=== Secure Settings
+
+Some settings are sensitive, and relying on filesystem permissions to protect
+their values is not sufficient. For this use case, Kibana provides a
+keystore, which may be password protected, and the `kibana-keystore`
```

Comment:
thoughts on saying this only obfuscates for now?

Reply:
I am going to just remove that part. I borrowed all the verbiage from https://github.com/elastic/elasticsearch/blob/master/docs/reference/setup/secure-settings.asciidoc

Reply:
8f6ae88fa215e758166c1cfc4771c2b60a7ee383

Reply:
@jaymode, the ES documentation mentions that the keystore can be password protected. While I feel it's best to omit that for now, should we note this is obscuring the password for now?